genny/ci/templates/-dot-circleci/config.yml.tmpl

```diff
@@ -0,0 +1,62 @@
+version: 2
+jobs: 
+  build:
+    docker:
+      - image: circleci/golang:1.12
+      {{ if eq .opts.DBType "postgres" -}}
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: {{.opts.App.Name.File}}_test
+      {{- end }}
+
+    environment:
+      TEST_RESULTS: /tmp/test-results
+
+    steps:
+      - checkout
+      - run: mkdir -p $TEST_RESULTS
+
+      - restore_cache:
+          keys:
+            - v1-pkg-cache
+
+      - run: go get github.com/jstemmer/go-junit-report
+
+      {{ if eq .opts.DBType "postgres" -}}
+      - run: go get github.com/lib/pq
+      - run: go get github.com/mattes/migrate
+      - run:
+          name: Waiting for Postgres to be ready
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for Postgres && exit 1
+      {{- end }}
+
+      - run:
+          name: Install Buffalo
+          command: |
+            go get github.com/gobuffalo/buffalo/buffalo
+            go get $(go list ./... | grep -v /vendor/)
+      - run:
+          name: Run unit tests
+          command: |
+            trap "go-junit-report <${TEST_RESULTS}/go-test.out > ${TEST_RESULTS}/go-test-report.xml" EXIT
```

Comment:
I didn't know about `go-junit-report` Thanks!. It comes very handy for some bamboo builds I have. does this add any functionality when used in CircleCI vs just running:
```
buffalo test -v
```

Reply:
It's my understanding that in order for CircleCI to display your test results in their dashboard they needed to be outputted at junit. If we don't care about that it's easy enough to skip.

Reply:
Got it. I used CircleCI in the past just by looking at the logs, Thanks for explaining.